app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -2,14 +2,14 @@
 
 .gem-c-metadata {
   @include govuk-text-colour;
-  @include govuk-font(16);
   @include responsive-bottom-margin;
   @include govuk-clearfix;
+  @include govuk-font(16);
```

Comment:
Interested - why does this one need to move, is there something different about this particular mixin?

Reply:
Good shout, turns out this change wasn't necessary, I must have got confused with the below warning

Reply:
Fine as it is though, no need to change 👍